### ISSUE TITLE
[Feature Request] Ability to control dark mode independently from the system

### ISSUE THREAD

**Issue by Gipetto** [Ranchero-Software/NetNewsWire#3165]
I like having the OS in Dark Mode but the apps in light mode, otherwise I get headaches reading light text on dark backgrounds.

I can override it via the terminal, but it would be nice to have something less fragile built in to the app.

``` sh
$  osascript -e 'id of app "NetNewsWire"'                                                                                
com.ranchero.NetNewsWire-Evergreen
$ defaults write com.ranchero.NetNewsWire-Evergreen NSRequiresAquaSystemAppearance -bool yes
```

This is related to #2179 but the work linked there doesn't look to have implemented the appearance suggestion.

**Comment by lylejantzi3rd** [comment]
@Gipetto Thank you so much for posting a workaround for this. I recently started using NetNewsWire and the white text on black background was giving me a headache. I would have hated to switch to another program just because of this minor issue.

**Comment by ironicmoka** [comment]
Thanks for the workaround!
I was hitting github to request that option too.

**Comment by microfloraa** [comment]
Would appreciate this option! I use Calibre reader in dark mode at all times even though my system is in light mode during the day, would be nice to do the same with NNW.

**Comment by seiz** [comment]
I would love to see that too. I generally have the system on Light Mode but torn some app to dark mode permanently (if they allow). One such app is Ivory and NNW would be one too.

Of course, I could whip up a theme, but that doesn't apply to the sidebars and rest of the App UI…

**Comment by forrie** [comment]
I agree, having a separate control for the UX/UI dark mode is better.  A lot of apps are following this direction, as there are many reasons why the system may not be able to use it.
